### ISSUE TITLE
Photoshop CC2015 Artboards are not cropped in Avocode

### ISSUE THREAD

**Issue by rychlis** [avocode/avocode#374]
Artboards with extraneous content outside will be enlarged in Avocode often causing them to overlap

**Comment by bolekkerous** [comment]
Unfortunately, Avocode doesn't support artboards correctly yet.

There's already a feature request on our UserVoice forum, which can be upvoted, so we know what to prioritize in the future:

https://avocode.uservoice.com/forums/277080-feature-voting/suggestions/9428469-support-photoshop-artboards

We'd advise to ungroup artboards in the design in the meantime. Once that is done, the design be displayed just fine.

**Comment by bolekkerous** [comment]
Hey - 
we’ve just released a new version - 2.19 - which I believe should fix this issue. 

Please update your app and it should hopefully work. If it doesn't, please try to reupload the design as well.

You can check our changelog for the full release notes:

https://avocode.com/changelog

If you still encounter this or any other issue, please reopen or create a new issue.

Best regards,
Bolek, Avocode Team
